storage/storage.go

```diff
@@ -0,0 +1,153 @@
+package storage
+
+import (
+	"database/sql"
+	"errors"
+	"log"
+	"os"
+	"path/filepath"
+
+	_ "github.com/mattn/go-sqlite3"
+
+	"final/task"
+)
+
+const Limit = 50
+
+type DB struct {
+	conn *sql.DB
+}
+
+func Createdatabase() (DB, error) {
+	appPath, err := os.Executable()
+	if err != nil {
+		log.Fatal(err)
+		return DB{conn: nil}, err
+	}
+	dbFile := filepath.Join(filepath.Dir(appPath), "scheduler.db")
+	_, err = os.Stat(dbFile)
+
+	var install bool
+	if err != nil {
+		install = true
+	}
+	// если install равен true, после открытия БД требуется выполнить
+	// sql-запрос с CREATE TABLE и CREATE INDEX
+	db, err := sql.Open("sqlite3", "scheduler.db")
+	if err != nil {
+		log.Fatal(err)
+		return DB{conn: nil}, err
+	}
+	if install {
+		createTableSql := `CREATE TABLE IF NOT EXISTS scheduler (
+			id INTEGER PRIMARY KEY AUTOINCREMENT,
+			date CHAR(8) NOT NULL DEFAULT "",
+			title VARCHAR(128) NOT NULL DEFAULT "",
+			comment TEXT NOT NULL DEFAULT "",
+			repeat VARCHAR(128) NOT NULL DEFAULT ""
+			);
+			CREATE INDEX IF NOT EXISTS scheduler_date ON scheduler (date);`
+		_, err = db.Exec(createTableSql)
+		if err != nil {
+			log.Fatal(err)
+			return DB{conn: nil}, err
+		}
+		return DB{conn: db}, nil
+	}
+	return DB{conn: db}, nil
+}
+
+func (db *DB) Addtasktodb(task task.Task) (int64, error) {
+	query := `INSERT INTO scheduler (date, title, comment, repeat) VALUES (?, ?, ?, ?)`
+	res, err := db.conn.Exec(query, task.Date, task.Title, task.Comment, task.Repeat)
+	if err != nil {
+		return 0, errors.New("Ошибка добавления задачи")
```

Comment:
Так ты теряешь саму ошибку и долго будешь разбираться, что случилось (коннект в БД не работает или ошибка в запросе)

Лучше оборачивать ошибку
```suggestion
		return 0, fmt.Errorf("Ошибка добавления задачи: %w", err)
```